src/js/sentry.ts

```diff
@@ -1,4 +1,5 @@
 import * as Sentry from '@sentry/browser';
+import { ChromeUser } from './types';
```

Comment:
Could be better to indicate we're importing `type`. Based on the doc, `import type` will be completely removed once transpiled.
```suggestion
import type { ChromeUser } from './types';
```

Reply:
We now import it form the types package